src/Styleguide/Components/SelectedExhibitions.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { SFC } from "react"
+import { Flex } from "../Elements/Flex"
+import { Box, BorderBox } from "../Elements/Box"
+import { Sans } from "@artsy/palette"
+
+type Year = string
+
+export interface Exhibition {
+  year: Year
+  show: string
+  gallery: string
+}
+
+type GroupedExhibitions = [Year, Exhibition[]]
+
+const groupExhibitions = (exhibitions: Exhibition[]): GroupedExhibitions[] => {
+  const exhibitionsByYear = exhibitions.reduce((acc, curr) => {
+    const { year } = curr
+    acc[year] = [].concat(acc[year], curr)
+    return acc
+  }, {})
+  return Object.keys(exhibitionsByYear)
+    .reverse()
+    .map(year => [year, exhibitionsByYear[year]] as GroupedExhibitions)
+}
+
+export interface ExhibitionsHeadlineProps {
+  exhibitionCount: number
+  expanded: boolean
+  collapsible: boolean
+}
+const ExhibitionsHeadline: SFC<ExhibitionsHeadlineProps> = props => (
+  <Flex justifyContent="space-between">
+    <Sans size="2" weight="medium" mb={3}>
+      {props.exhibitionCount < 3 ? "Exhibitions" : "Selected exhibitions"}
+      {props.collapsible && !props.expanded
+        ? ` (${props.exhibitionCount})`
+        : ""}
+    </Sans>
+    {props.collapsible &&
+      !props.expanded && (
+        <Sans size="2" color="black60">
+          Show
+        </Sans>
+      )}
+  </Flex>
+)
+
+export interface ExhibitionYearListProps {
+  year: Year
+  exhibitions: Exhibition[]
+}
+const ExhibitionYearList: SFC<ExhibitionYearListProps> = props => (
+  <Flex>
+    <Sans size="2">{props.year}</Sans>
+    <Flex flexDirection="column">
+      {props.exhibitions.filter(e => e).map(exhibition => (
```

Comment:
Any need for `filter` here, or just defensive coding?

Reply:
There were actually some empty values for whatever reason. I should probably know that reason. I do not.

Reply:
I'll take another look..